gordo_components/builder/build_model.py

```diff
@@ -314,13 +320,39 @@ def set_seed(self, seed: int):
         np.random.seed(seed)
         random.seed(seed)
 
+    @staticmethod
+    def build_split_dict(X: pd.DataFrame, tssplit: TimeSeriesSplit) -> dict:
+        """
+        Get dictionary of cross-validation training dataset split metadata
+
+        Parameters
+        ----------
+        X: pd.DataFrame
+            The training dataset that will be split during cross-validation.
+        tssplit: sklearn.model_selection.TimeSeriesSplit
+            The cross-validation object that returns train, test indices for splitting.
+
+        Returns
+        -------
+        split_metadata: Dict[str,Any]
+            Dictionary of cross-validation train/test split metadata
+        """
+        split_metadata: Dict[str, Any] = dict()
+        for i, (train_ind, test_ind) in enumerate(tssplit.split(X)):
+            for name, ind in zip(["train", "test"], [train_ind, test_ind]):
+                for pos, idx in zip(["start", "end"], [0, -1]):
+                    split_metadata.update(
+                        {f"fold_{i+1}_{name}_{pos}": X.index[ind[idx]]}
+                    )
+                split_metadata.update({f"fold_{i+1}_n_{name}": len(ind)})
```

Comment:
So the method below this uses hyphens, maybe this should to since that's the better practice when things are destined for json?